.github/workflows/phpunit-tests.yml

```diff
@@ -39,9 +39,14 @@ jobs:
                     - '8.1'
                     - '8.2'
                 multisite: [false, true]
+                wordpress: ['WordPress/WordPress']
+                include:
+                    - php: '7.4'
+                    - wordpress: 'WordPress/WordPress#6.0-branch'
```

Comment:
Note that this is also the the value that could be used to label the job in CI, which isn't very nice, so we might want to consider changing it.

The advantage of using the `x.y-branch` is that we don't have to compute the latest .patch version in that series (would be 6.0.3 here).

In a follow-up, we could consider computing the previous WP series automatically (i.e. figure out that 6.1 is the current series, and that thus, 6.0 is the previous one).